### ISSUE TITLE
Add LBGAT on CIFAR-10

### ISSUE THREAD

**Issue by jiequancui** [fra31/auto-attack#36]
**Paper**: {Learnable Boundary Guided Adversarial Training; https://arxiv.org/pdf/2011.11164.pdf}

**Venue**: {if applicable, the venue where the paper appeared}

**Dataset and threat model**: {CIFAR-10, L-inf and epsilon 0.031}

**Code**: { [code](https://github.com/FPNAS/LBGAT)}

**Pre-trained model**: [LBGAT0-wideresnet-34-10](https://drive.google.com/file/d/1JufuOi5szINv2oSZ7iNnFrKzNaXiLG1-/view?usp=sharing), [LBGAT0-wideresnet-34-20](https://drive.google.com/file/d/1RpqN3QwD7-QNIFGQfqcFG9FOQTAg4LOm/view?usp=sharing)

**Log file**: {[LBGAT0-wideresnet-34-10](https://drive.google.com/file/d/1TLCmiG79otcsECsoxGq2j6_5i9nhKfTf/view?usp=sharing), [LBGAT0-wideresnet-34-20](https://drive.google.com/file/d/1c13_h731jJsnxeIMOD2-0FrWZLEpbgRz/view?usp=sharing)}

**Additional data**: {no}

**Clean and robust accuracy**: {88.22/52.86, 88.70/53.57}

**Architecture**: {wideresnet-34-10; wideresnet-34-20}

**Description of the model/defense**: {Our method aims to enhance model robustness while preserving high natural accuracy. }

**Comment by fra31** [comment]
Hi,

thanks for submission! I've added the models.

**Comment by FPNAS** [comment]
Very Thanks.

**Comment by jiequancui** [comment]
Hi,

Our paper "Learnable boundary guided adversarial training" is to appear in ICCV2021. Can you help update the "Venue status" for our LBGAT models in Robust Bench?
